mod/wikirate/set/right/contributed_metrics.rb

```diff
@@ -1,19 +1,20 @@
+def contribution_count
+  @cc ||= Card.search :type_id=>MetricID, :or=>
+                      {
+                       :left=>cardname.left,
```

Comment:
since this `:left=>cardname.left` is in the or clause, won't this find every metric that has a plus card?